### ISSUE TITLE
Web: cannot copy/cut/paste in webviews using keyboard

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#106383]
Steps to Reproduce:

1. open https://vscode-web-test-playground.azurewebsites.net/?enter=true
2. open a markdown file
3. add `<input>` element
4. open preview
5. try to cut/copy/paste in the input field

=> 🐛 it does not work

I think this only reproduces in a built version of VSCode web, not running out of sources. Using bisect I found the commit range to be this: https://github.com/microsoft/vscode/compare/e7920dce7bfd0b707ebfc0a5379c6edd2233e475...3bc3f95dbc0434fd88bc3ae99b8aca8b281fc622

My guess would be: https://github.com/microsoft/vscode/commit/7e5937d684eb1dcbe5f199eaf81152411e1ef7e1

**Comment by mjbvz** [comment]
It seems to work for me when building from main with `yarn web`. Not sure what is different about that 

In both cases however, we are not using our custom copy command. Here's what happens:

1. User presses cmd+c in webview
1. We intercept the keypresses in the webview and [send them back to VS Code](https://github.com/microsoft/vscode/blob/80b1a775d691612ce8334238d786a922d55e3544/src/vs/workbench/contrib/webview/browser/baseWebviewElement.ts#L331)
1. We then handle the keypress in our keybindings service.
1. However on web, the copy command lacks an actual keybinding: https://github.com/microsoft/vscode/blob/f9d74e9be9562659ad3d0f14d47e484663ad297a/src/vs/editor/contrib/clipboard/clipboard.ts#L73
1. This means we never resolve a keybinding for `cmd+c`

@alexdima Any thoughts on how best to handle this case? Does the comment in `clipboard.ts` still apply?

**Comment by alexdima** [comment]
@mjbvz Yes, that is intentional. In browsers we don't want to ship with keybindings for `cmd+c`,`cmd+v` or `cmd+x`. We want browsers to execute their default action, we don't call `preventDefault()` and then browsers will invoke copy/paste/cut events **without a security prompt**. If we were to use a keybinding, we would have to go through `clipboardService`, which goes through the navigator clipboard API, which is not fully implemented in Firefox (so cmd+v will no longer work there at all) and which prompts in other browsers. 

Long story short, my recommendation is that on the web you also do not capture `cmd+c`/`cmd+v`/`cmd+x` and allow the browser to generate copy/paste/cut events and add listeners for these events.

**Comment by mjbvz** [comment]
Thanks for the background @alexdima! 

I previously added special logic that calls `preventDefault` for copy/paste: 7e5937d684eb1dcbe5f199eaf81152411e1ef7e1

We need this on desktop to prevent double pasting but I will remove the call to `preventDefault` on web

**Comment by bpasero** [comment]
Thanks, verified in https://vscode-web-test-playground.azurewebsites.net/?enter=true with todays build.
